CONTRIBUTING.md

```diff
@@ -77,7 +77,7 @@ When you submit a new code example to us, we strongly encourage you to include t
   of standard error/exception handling.  
 * **Don't include personal account data, keys, or IDs in your examples**. Code should obtain access keys from the standard AWS SDK credentials and configuration files, use environment variables or external data files, or query the user for this information.
 * **Format code lines to 80 characters wherever possible**. Long lines can often spill off the side of the screen in the PDF versions of the documentation, making the code unreadable. If your code includes long text strings, consider breaking these into smaller chunks and concatenating them.
-* **Use spaces, not tabs, for indentation**. Tabs are variable length in most editors, but will usually render as 8 characters wide in printed documentation. *Always use spaces* to ensure consistent formatting in printed code. You can ignore this rule for makefiles, which might *require* the use of tabs. But these are typically used only for building examples, and aren't  included in documentation.
+* **Use 4 spaces, not tabs, for indentation**. Tabs are variable length in most editors, but will usually render as 8 characters wide in printed documentation. *Always use 4 spaces* to ensure consistent formatting in printed code. You can ignore this rule for makefiles, which might *require* the use of tabs. But these are typically used only for building examples, and aren't  included in documentation.
```

Comment:
We might want to say 4 spaces unless the target language has a different convention. For example, it's 2 spaces for Ruby (https://rubystyle.guide/#spaces-indentation).